recipes/psyclone/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "psyclone" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/psyclone-{{ version }}.tar.gz
+  sha256: 7b369353942358afcb93b199ef2b11116d756cf9d671667ca95fa83fb31f0355
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - pyparsing
+    - fparser >=0.2.0
+    - configparser
+    - jsonschema
+    - sympy
+    - jinja2
+    - termcolor
+    - python-graphviz
+
+test:
+  imports:
+    - psyclone
+  commands:
+    - pip check
+    - psyclone --version  # [not win] CLI not available on Windows
```

Comment:
Is the rest of the library OK on Windows? See it has scripts: https://github.com/stfc/PSyclone/blob/e755f6bfd8cf7f2103f331ab834ae05c57989a06/setup.py#L174

Reply:
Thanks for the review.

I don't think there's anything preventing it from running on Windows other than it using script files rather than entry points, though I don't know if anyone runs it from Windows.

I would be happy skipping packaging for Windows until this gets fixed upstream, I can't find in the conda-forge documentation how to do that though.